app/src/studio/uuid.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+#include <zephyr/bluetooth/uuid.h>
+
+#define ZMK_BT_STUDIO_UUID(num) BT_UUID_128_ENCODE(num, 0x0196, 0x6107, 0xc967, 0xc5cfb1c2482a)
```

Comment:
I see this was also used in the split uuid, but some comments would be nice for these magic numbers, if they can't be made a `#define`. Or maybe it is completely arbitrary?

Reply:
They are truly arbitrary. Other than one "reserved" 128-bit "group", vendor specific 128-bit UUIDs should just be generated randomly.